src/mn_menu.c

```diff
@@ -1798,7 +1798,7 @@ static void M_SizeDisplay(int choice)
         default:
             break;
     }
-    screenblocks = BETWEEN(3, 12, screenblocks);
+    screenblocks = BETWEEN(3, 13, screenblocks);
```

Comment:
What if a PWAD contains an SBARDEF lump that does not provide the extra HUD?

Reply:
Should be OK. We will show the last available status bar.